en/landing.ftl

```diff
@@ -0,0 +1,160 @@
+bundle-banner-plan-header-2 = Your 1-year plan includes:
+bundle-banner-plan-modules-email-masking-2 = Email masking
+bundle-banner-plan-modules-phone-masking-2 = Phone masking
+bundle-banner-plan-modules-mozilla-vpn-2 = { -brand-name-mozilla-vpn }
+bundle-banner-cta-2 = Get { -brand-name-mozilla-vpn } + { -brand-name-relay }
+bundle-banner-alt-2 = { -brand-name-mozilla-vpn } and { -brand-name-relay }
+# Variables:
+#   $days_guarantee (string) - the number of days for money-back guarantee. Examples: 30, 90
+bundle-banner-money-back-guarantee-2 = { $days_guarantee }-day money-back guarantee for first-time subscribers
+# Variables:
+#   $num_vpn_servers (string) - the number of VPN servers. Examples: 400, 500, 600
+bundle-feature-one-2 = More than { $num_vpn_servers } servers
+# Variables:
+#   $num_vpn_countries (string) - the number of VPN available countries. Examples: 30, 40, 50
+bundle-feature-two-2 = More than { $num_vpn_countries } countries
+bundle-feature-three-2 = Fast and secure network
+
+## HOW IT WORKS SECTION 
+
+how-it-works-section-title = How { -brand-name-relay } email & phone masking works
+# Item 1
+how-it-works-section-extension-headline = Use { -brand-name-relay } email masks and phone masks everywhere
+how-it-works-section-extension-body = Get secure, random email masks and a unique, masked phone number to use any time a website, app, store, or restaurant asks for your information.
+# Item 2
+how-it-works-section-forward-headline = We’ll forward emails, phone calls, and texts to you
+how-it-works-section-forward-body = Rest assured senders will never know your real email address or real number. You can even reply to texts and emails without sharing your real identity.
+# Item 3
+how-it-works-section-manage-headline = Manage your masks from your { -brand-name-relay } dashboard
+how-it-works-section-manage-body = Sign in to your Relay dashboard to create new masks, label existing masks, and  block calls, texts, or emails from masks that get too much spam.
+
+## PRICING MATRIX SECTION
+
+plan-matrix-offer-title = Choose a level of protection that’s right for you
+# Top Row Labels
+# Variables:
+#   $savings (string) - the percentage saved (including % symbol) for a given plan. Examples: 50%, 70%
+plan-matrix-offer-body = { -brand-name-firefox-relay } protects your identity and your inbox with unique email masks. Add on even more security with { -brand-name-relay } phone masking, and save { $savings } by adding { -brand-name-vpn } protection.
+plan-matrix-heading-features = Features
+plan-matrix-heading-plan-free = Limited email protection
+plan-matrix-heading-plan-premium = Email protection
+plan-matrix-heading-plan-phones = Email & phone protection
+plan-matrix-heading-plan-bundle = Add VPN protection
+plan-matrix-heading-plan-bundle-alt = Get { -brand-name-relay-premium } and { -brand-name-mozilla-vpn }
+# Feature Breakdowns
```

Comment:
This comment is only viewable in Pontoon for the first string below

Reply:
You need to check with the other devs. I am sure there are lots of elements in the page that are not meant for l10n to look at. There must be a way to exclude it.